src/test/kotlin/com/roadlink/tripservice/usecases/trip_search/SearchTripTest.kt

```diff
@@ -30,104 +46,102 @@ internal class SearchTripTest {
 
     @Test
     fun `given no section exists then should return empty list`() {
+        // GIVEN
+        val departure = LocationFactory.avCabildo_4853()
+        val arrival = LocationFactory.avCabildo_20()
+        val at = InstantFactory.october15_12hs()
+        circleSearchAreaRadius = circleSearchAreaRadius(departure, arrival)
+        givenNextSections(from = departure, at = at, nextSections = emptySet())
+
+        // WHEN
         val result = searchTrip(
             SearchTrip.Input(
-            departure = LocationFactory.avCabildo_4853(),
-            arrival = LocationFactory.avCabildo_20(),
-            at = InstantFactory.october15_12hs(),
+            departure = departure,
+            arrival = arrival,
+            at = at,
         ))
 
+        // THEN
         assertTrue(result.isEmpty())
     }
 
     @Test
     fun `given exists a trip plan between the given departure and arrival then should return it`() {
-        inMemorySectionRepository.save(SectionFactory.avCabildo())
-
-        val result = searchTrip(
-            SearchTrip.Input(
-            departure = LocationFactory.avCabildo_4853(),
-            arrival = LocationFactory.avCabildo_20(),
-            at = InstantFactory.october15_12hs(),
-        ))
-
-        assertEquals(listOf(TripSearchPlanFactory.avCabildo()), result)
-    }
-
-    @Test
-    fun `given exists a trip plan but not between the given departure and arrival then should return empty list`() {
```

Comment:
por que se borro este test? el comportamiento tiene que seguir siendo el mismo para estos casos no?

Reply:
Porque esto antes usaba el in memory, y ahora tenemos un mock. La responsabilidad de cuales son las secciones cuyo departure estan cerca del punto de busqueda se la delegamos a la db, entonces aca al mockear era siempre decir que retorne una lista vacia el metodo, por lo cual todos los test eran lo mismo, era ver como se comporta este codigo al recibir una lista vacia como respuesta.